### ISSUE TITLE
`cci flow run dev_org` errors out on `load_sample_data`

### ISSUE THREAD

**Issue by allisonletts** [SFDO-Tooling/CumulusCI#3505]
### Describe the bug

The `load_sample_data` task that was added to the end of `cci flow run dev_org` references a file that may not exist in old CCI projects. The flow should handle that file not existing, or provide specific instructions about how to fix it.

To resolve, I added these lines to my cumulusci.yml file:
```yml
flows:
    config_dev:
        steps:
            90:
                task: None
```
I do have a folder at datasets/default, but it's empty (except for a .gitkeep) -- there isn't a default recipe.

### Reproduction steps

1. Run `cci flow run dev_org` on a repo with a default org
2. Note that the flow runs until it errors out on `load_sample_data`

### Your CumulusCI and Python versions

CumulusCI version: 3.71.0 (C:\Users\AllisonHLetts\.local\bin\cci)
Python version: 3.10.8 (C:\Users\AllisonHLetts\.local\pipx\venvs\cumulusci\Scripts\python.exe)

### Operating System

Windows 10 Enterprise

### Windows environment

PowerShell

### CumulusCI installation method

pipx

### Error Gist

https://gist.github.com/allisonletts/af3c5896bb7c6e22d7d5799ac6846f2a

**Comment by prescod** [comment]
Thanks for the report Allison!

The fix for this is in PR 3502
<https://github.com/SFDO-Tooling/CumulusCI/pull/3502>

**Comment by jstvz** [comment]
Fixed in #3502
